Sources/Danger/Plugins/SwiftLint/ShellExecutor.swift

```diff
@@ -62,6 +45,27 @@ internal class ShellExecutor {
         let stderrData = stdout.fileHandleForReading.readDataToEndOfFile()
         let stderrString = String(data: stderrData, encoding: String.Encoding.utf8)!
 
-        throw SpawnError.commandFailed(exitCode: task.terminationStatus, stdout: stdoutString, stderr: stderrString, task: task)
+        throw SpawnError.commandFailed(exitCode: task.terminationStatus,
+                                       stdout: stdoutString,
+                                       stderr: stderrString,
+                                       task: task)
+    }
+
+    private func makeTask(for command: String, with arguments: [String], environmentVariables: [String: String]) -> Process {
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/Danger/Plugins/SwiftLint/ShellExecutor.swift;
  Line: 54;
-->

- :warning: Line should be 120 characters or less: currently 125 characters (`line_length`)